### PR TITLE
FPv2: log responses from data collection queries

### PR DESCRIPTION
**Pull request by sshane**
Marks responses to queries added for logging, so we don't fingerprint on them later. Fixes a bug where this Civic 2022 always fuzzy fingerprints due to data collection response replacing missing real response (not supported for some reason by the radar on this specific car): https://github.com/commaai/openpilot/pull/27126

### CODE REVIEW

**Comment by sshane** [comment]
This PR will add the `logging` field to all `CarFw` items in the `carFw` list that:

- Are returned from a data-collection-only request
- Correspond to an `extra_ecu` (ECUs that are added to just gather their FW versions)

File: selfdrive/car/fw_versions.py [hunk 1]
```diff
@@ -232,15 +231,19 @@ def get_fw_versions(logcan, sendcan, query_brand=None, extra=None, timeout=0.1,
...
-      for ecu_type, addr, sub_addr in c.keys():
+    for candidate, ecu in brand_versions.items():
+      for ecu_type, addr, sub_addr in ecu.keys():
         a = (brand, addr, sub_addr)
         if a not in ecu_types:
           ecu_types[a] = ecu_type
 
+        if a not in logging_addrs and candidate == "debug":
+          logging_addrs.append(a)
+
```
Comment (sshane) [inline selfdrive/car/fw_versions.py:246]: continues an old convention, want to refactor eventually

**Comment by sshane** [comment]
@Ktrtkd are you still experiencing issues?

**Comment by Ktrtkd** [comment]
> @Ktrtkd are you still experiencing issues?

For this topic no.

**Pull request merged**